docker/Dockerfile

```diff
@@ -220,7 +225,14 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # Install vllm wheel first, so that torch etc will be installed.
 RUN --mount=type=bind,from=build,src=/workspace/dist,target=/vllm-workspace/dist \
     --mount=type=cache,target=/root/.cache/uv \
-    uv pip install --system dist/*.whl --verbose
+    uv pip install --system dist/*.whl --verbose \
+    --extra-index-url https://download.pytorch.org/whl/test/cu128 \
+    --index-strategy unsafe-best-match
+
+# TESTING: install xformers from source to test 2.7.0 final RC
+RUN --mount=type=cache,target=/root/.cache/uv \
```

Comment:
If the intention is that the we are just going to wait till xformers releases a 12.8 compat whl though ignore above

Reply:
IMO, I think we can just build the package from source for CI, then switch to the official xformers package once it's ready

Reply:
How long does this take to compile xformers? If it's too long I don't want to slow down our CI time for this.

Reply:
Let me dig out the number for this once the build finish.  Without caching, it would be significant from what I see locally, but let's see what it takes to build once this is cached,